File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_vip_recharge_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusVipRechargeVO:
    """VIP充值记录Value Object"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    uuid: str = None
    user_id: int = None
    vip_level_id: int = None
    amount: float = None
    point_amount: int = None
    recharge_type: str = None
    recharge_time: str = None
    transaction_no: str = None
    status: int = None
    remark: str = None

File: sdkwork-backend-sdk-go/types/plus_vip_recharge_vo.go
package types

// VIP充值记录Value Object
type PlusVipRechargeVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	UserId int `json:"userId"`
	VipLevelId int `json:"vipLevelId"`
	Amount float64 `json:"amount"`
	PointAmount int `json:"pointAmount"`
	RechargeType string `json:"rechargeType"`
	RechargeTime string `json:"rechargeTime"`
	TransactionNo string `json:"transactionNo"`
	Status int `json:"status"`
	Remark string `json:"remark"`
}
